build.sbt

```diff
@@ -12,7 +12,7 @@ licenses := Seq("Apache 2.0" -> url("https://raw.githubusercontent.com/amplab/ke
 
 homepage := Some(url("http://keystone-ml.org"))
 
-scalaVersion := "2.10.4"
+scalaVersion := "2.10.6"
```

Comment:
Yeah its more a question of should we publish maven artifacts for 2.11 and 2.10 - the ml-matrix one does that right now